test/gui/shared/steps/steps.py

```diff
@@ -143,31 +151,88 @@ def isItemSynced(type, itemName):
     if not socketConnect.read_socket_data_with_timeout(0.1):
         return False
     for line in socketConnect.get_available_responses():
-        if line.startswith('STATUS:OK') and line.endswith(itemName):
+        if line.startswith(status) and line.endswith(itemName):
             return True
         elif line.endswith(itemName):
             return False
 
 
-def isFolderSynced(folderName):
-    return isItemSynced('FOLDER', folderName)
+def folderHasSyncStatus(folderName, status):
+    return hasSyncStatus('FOLDER', folderName, status)
 
 
-def isFileSynced(fileName):
-    return isItemSynced('FILE', fileName)
+def fileHasSyncStatus(fileName, status):
+    return hasSyncStatus('FILE', fileName, status)
 
 
-def waitForFileToBeSynced(context, filePath):
-    waitFor(
-        lambda: isFileSynced(sanitizePath(filePath)),
-        context.userData['clientSyncTimeout'] * 1000,
+def waitForFileOrFolderToHaveSyncStatus(
+    context, resource, resourceType, status=SYNC_STATUS['OK']
+):
+    timeout = context.userData['clientSyncTimeout'] * 1000
+    if status == SYNC_STATUS['SYNC']:
+        timeout = 5000
```

Comment:
Can you add a little comment, to justify the timeout?

Reply:
need to figure the better way to check if sync is started or not